README.md

```diff
@@ -39,12 +32,45 @@ def application do
 end
 ```
 
+## If you'd like to mount VerkWeb on another Endpoint:
+
+
+```elixir
+defmodule MyApp.Endpoint do
+  use VerkWeb.Mount, path: "/verk"
+  ...
+end
+```
+
+That should be it! :)
```

Comment:
I see. It's funny, how differently we see our use cases and try to simplify **our** way of using the app.

So, if using as standalone is zero-config, then using with mounting would require configuration in 2 places: 
- main app endpoint + config.exs 

If we keep current settings, you'd have to adjust only 1 file for any of approaches... So: 
- either 2 - 0 or 1 - 1. It might seem like a trivial thing, but keeping mental configuration overhead balanced goes a long way. 
How likely it is that the user will keep the default port 4000 for VerkWeb? 

Don't get me wrong, zero-config approaches are cool, but the probability of our user not overriding the defaults is really low... So, we'd still have to explain in the readme how to configure it anyways. 

In the end it's your decision. If you think that's way, I'll change the default server to true and the user of the mounting plug will have to turn it off in config.exs + mount it in endpoint. 

Should I change the default to  `server: true`?